ruby_programming/computer_science/project_recursion.md

```diff
@@ -5,7 +5,7 @@ The [Fibonacci Sequence](http://en.wikipedia.org/wiki/Fibonacci_number), which s
 ### Assignment 1
 
 <div class="lesson-content__panel" markdown="1">
-  1. Write a method `#fibs` which takes a number and returns that many members of the fibonacci sequence.  Use iteration for this solution.
+  1. Using iteration, write a method `#fibs` which takes a number and returns that many members of the fibonacci sequence. It should return an array.
```

Comment:
```suggestion
  1. Using iteration, write a method `#fibs` which takes a number and returns an array containing that many numbers from the fibonacci sequence.
```

Reply:
That makes perfect sense, thank you. I made the change.